DQM/HLTEvF/plugins/HLTObjectMonitorHI.cc

```diff
@@ -0,0 +1,916 @@
+// -*- C++ -*-
+//
+// Package:    QM/HLTObjectMonitorHI
+// Class:      HLTObjectMonitorHI
+//
+/**\class HLTObjectMonitorHI HLTObjectMonitorHI.cc DQM/HLTEvF/plugins/HLTObjectMonitorHI.cc 
+
+ Description: [one line class summary]
+
+ Implementation:
+     [Notes on implementation]
+*/
+//
+// Original Author:  Geonhee Oh
+//         Created:  Sun, 29 Oct 2018 17:00:00 GMT
+//
+//
+
+
+// system include files
+#include <memory>
+#include <sys/time.h>
+#include <cstdlib>
+
+// user include files
+#include "DataFormats/Common/interface/TriggerResults.h"
+#include "DataFormats/HLTReco/interface/TriggerObject.h"
+#include "DataFormats/HLTReco/interface/TriggerEvent.h"
+#include "DataFormats/HLTReco/interface/TriggerTypeDefs.h"
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "HLTrigger/HLTcore/interface/HLTConfigProvider.h"
+
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "DQMServices/Core/interface/MonitorElement.h"
+#include "DQMServices/Core/interface/DQMEDAnalyzer.h"
+
+#include <TMath.h>
```

Comment:
I prefer to have the ROOT includes before the CMSSW ones, but it's mostly a matter of taste